privacy-snapshot.json

```diff
@@ -29,6 +29,8 @@
   "github.com",
   "goerli.infura.io",
   "lattice.gridplus.io",
+  "linea-mainnet.infura.io",
+  "linea-sepolia.infura.io",
```

Comment:
Had to add these for the new added e2e because after multichain it is now also making calls to linea mainnet and linea sepolia